bee-message/src/output/foundry.rs

```diff
@@ -227,7 +227,7 @@ impl FoundryOutput {
     ///
     #[inline(always)]
     pub fn address(&self) -> &Address {
-        // A FoundryOutput must have a AddressUnlockCondition.
+        // A FoundryOutput must have an AddressUnlockCondition.
```

Comment:
Maybe use doc comments here if we're touching it already.

Reply:
There are plenty of instances of missing link tbh, if you allow me I'll do it as part of the bee-message documentation PR

Reply:
Sure, that's why I already approved. :)